ocean_lib/data_provider/data_service_provider.py

```diff
@@ -655,7 +662,6 @@ def _prepare_compute_payload(
             )
         else:
             payload["algorithmMeta"] = algorithm_meta
-
```

Comment:
a bit of a nitpick here: please add this line back, keeping only relevant changes.